lib/cob_index/macros/custom.rb

```diff
@@ -588,7 +599,8 @@ def lookup_hathi_bib_key
       def lookup_hathi_bib_key_in_files(oclc_num)
         base_path = __dir__ + "/../../hathi_data"
         trailing_digit = oclc_num[-1]
-        `grep -o '^.*,#{oclc_num}$' #{base_path}/trailing_#{trailing_digit}.csv`.split(",").first
+        line = `egrep -o -m 1 '^.+,.+,#{oclc_num}$' #{base_path}/trailing_#{trailing_digit}.csv`.split(",")
+        { bib_key: line[1], access: line[0] }.to_json unless line.empty?
```

Comment:
@bibliotechy this will return `nil` when `line.empty?` but the `lookup_hathi_bib_key` doesn't check for that ... i.e. `acc < nil ` is a possibility.  Is that something we want to avoid?  Especially if we plan to define this display field as json in the solr config?

Reply:
@dkinzer pretty sure traject automatically removes the `nil` values, so it just becomes a no-op on the display side. 
https://github.com/tulibraries/cob_index/blob/4bbcb99b75355e4fb3eadc960ddc47d75032d1ca/spec/cob_index/macros/custom_spec.rb#L1588-L1592